src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PersonDetailsService.kt

```diff
@@ -92,6 +92,13 @@ internal class PersonDetailsService(
     )
   }
 
+  private fun isNoFixedAbode(it: uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.ndelius.Address): Boolean {
+    val postcodeNoWhiteSpace = it.postcode
+      ?.filter { postcodeCharacter -> !postcodeCharacter.isWhitespace() }
+    val noFixedAbode = postcodeNoWhiteSpace == "NF11NF" || it.noFixedAbode == true
```

Comment:
I wonder if someone would ever enter in in delius as lowercase, might be worth uppercasing the value before doing this check?

Reply:
Neat point :)! Done!